iTechArt.CinemaWebApp.API/Application/Services/AccountService.cs

```diff
@@ -33,44 +33,38 @@ public class AccountService
             _mapper = mapper;
             _config = configuration;
         }
+
         public async Task<Response<LoginResponse>> LoginAsync(LoginRequest request)
         {
             var user = await _context.Users.FirstOrDefaultAsync(user => user.Email == request.Email);
 
-            if(user == null)
-            {
-                return new Response<LoginResponse>($"No Accounts Registered with {request.Email}.");
-            }
+            if (user == null) return new Response<LoginResponse>($"No Accounts Registered with {request.Email}.");
 
-            var validatePassword = BCrypter.EnhancedVerify(request.Password, user.PasswordHash);
-            
-            if(!validatePassword)
-            {
-                return new Response<LoginResponse>($"Invalid Credentials for {request.Email}.");
-            }
+            var validatePassword = BCrypter.EnhancedVerify(request.Password,
+                user.PasswordHash);
```

Comment:
Either use one line (if it doesn't exceed 120 characters) or the following style:
```suggestion
            var validatePassword = BCrypter.EnhancedVerify(
                request.Password,
                user.PasswordHash
            );
```